CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ project(rocprim LANGUAGES CXX)
 # CMake modules
 list(APPEND CMAKE_MODULE_PATH
   ${CMAKE_CURRENT_SOURCE_DIR}/cmake
-  ${HIP_PATH}/cmake /opt/rocm/hip/cmake # FindHIP.cmake
+  ${HIP_PATH}/cmake ${CMAKE_INSTALL_PREFIX}/hip/cmake /opt/rocm/hip/cmake # FindHIP.cmake
```

Comment:
Please change to avoid using the CMAKE_INSTALL_PREFIX.

Reply:
You are suggesting to remove CMAKE_INSTALL_PREFIX and use ROCM_PATH?